### PR TITLE
Responsive UI and known JS bug fix

### PR DESCRIPTION
**Pull request by subhranshuchoudhury**
* Responsive UI using CSS Media query.
* Bootstrap for Accordion
* trim() used in script.js, we get value from button, so some browsers add spaces in button values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ To run the application you need to clone the project and run the HTML file.
 ![image](https://user-images.githubusercontent.com/72201530/193838132-4e22f252-ce18-42ef-8066-b1e392d784a8.png)
 
 ## 🔗 Links
+
```
Comment (ppfeiler) [inline README.md:30]: this should not be neccessary.

File: index.html [hunk 2]
```diff
@@ -1,61 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
```
Comment (ppfeiler) [inline index.html:6]: please dont use a cdn for this. Just at npm and add bootstrap as an dependency.

File: index.html [hunk 3]
```diff
@@ -1,61 +1,144 @@
...
     </div>
-</div>
-<script src="script.js"></script>
-<br></br>
-
-<footer class="end">
-    <p class="love">Made with <span style="color: #e25555;">&hearts;</span> by <a
-            href="https://twitter.com/Dhanush_Nehru"> Dhanush Nehru </a></p>
-</footer>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
```
Comment (ppfeiler) [inline index.html:130]: please dont use a cdn for this. Just at npm and add popperjs as an dependency.

File: index.html [hunk 4]
```diff
@@ -1,61 +1,144 @@
...
+        integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.min.js"
```
Comment (ppfeiler) [inline index.html:133]: please dont use a cdn for this. Just at npm and add bootstrap as an dependency.

File: script.js [hunk 5]
```diff
@@ -3,47 +3,47 @@ var buttons = document.getElementsByClassName("button");
 
 Array.prototype.forEach.call(buttons, function (button) {
   button.addEventListener("click", function () {
-    console.log(" Button text content", button.textContent);
+    console.log(" Button text content", button.textContent.trim());
```
Comment (ppfeiler) [inline script.js:6]: please extract `button.textContent.trim()` to a variable and use it on all the places.

`const buttonValue = button.textContent.trim();`

**Review by ppfeiler — COMMENTED** [review]
I am not sure if its a good idea to add bootstrap just because of the accordion. I am pretty sure that we can use a lot more of bootstraps features to create an responsiv layout.

File: script.js (hunk 5 above)
Comment (subhranshuchoudhury) [reply script.js:6]: That just a console log no need to assign a variable for that.

File: index.html (hunk 2 above)
Comment (subhranshuchoudhury) [reply index.html:6]: The base of the project is only a static HTML page. Convert it to a node js project kind of headache and not necessary. And all the cdn are cacheable so it is already available in the browser We don't need a get request for fetching pooper js or bootstrap css file.

File: script.js (hunk 5 above)
Comment (ppfeiler) [reply script.js:6]: And the other n times in this code block?

Comment (subhranshuchoudhury) [reply script.js:6]: Ok i got your concern.

File: README.md (hunk 1 above)
Comment (subhranshuchoudhury) [reply README.md:30]: I didnt get it the links are necessary i think 😃

**Review by DhanushNehru — APPROVED** [review]

**Comment by DhanushNehru** [comment]
Please recheck the comments here https://github.com/DhanushNehru/calculator/issues/1

**Comment by subhranshuchoudhury** [comment]
I think there should be a label hacktoberfest-accepted

**Pull request merged**